nixui/graphics/field_widgets.py

```diff
@@ -258,25 +230,27 @@ def validate_field(value):
         return isinstance(value, bool)
 
     def load_value(self, value):
+        if not self.validate_field(value):
+            value = False
```

Comment:
Maybe this would be better as `None` or the like? We currently don't handle field deletion yet, but I'd rather the field fallback to default instead of to `False`. This could be bad if e.g. it accidentally disables SSH on a remote server.

Reply:
This sets the default state for field widgets. It cannot be None because that isn't a valid `QCheckBox.checked` state.

In other words it is the first state the field enters into if moving from a state which cannot be tranferred (e.g. `Undefined` -> `bool`).

Does the line make sense in this context, or should it be changed somehow?